tests/functional/RemoteTargetLocatorTest.php

```diff
@@ -71,4 +72,86 @@ public function testActiveElement()
         $this->assertSame('input', $activeElement->getTagName());
         $this->assertSame('test_name', $activeElement->getAttribute('name'));
     }
+
+    /**
+     * Data provider for frame().
+     */
+    public function frameProvider()
```

Comment:
I would prefer this name (I will later rename all other providers to similar naming scheme `provide...()`)

```suggestion
    public function provideFrame()
```